blazor/ClientSideExample/ClientSideExample/ViewModels/CounterViewModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reactive;
+using System.Threading.Tasks;
+using ReactiveUI;
+
+namespace ClientSideExample.ViewModels
+{
+    public class CounterViewModel : ReactiveObject
+    {
+        private int currentCount;
+
+        public CounterViewModel()
+        {
+            Increment = ReactiveCommand.CreateFromTask(IncrementCount);
+        }
+
+        public int CurrentCount => currentCount;
```

Comment:
This seems to change, should this call RaiseAndSetIfChanged?

Reply:
It works without 

assume shrug-emoji here.

Reply:
I would just make it RaiseAndSetIfChanged, I suspect in WASM land this might break.

Reply:
Done.